source/features/profile-gists-link.tsx

```diff
@@ -41,15 +52,14 @@ async function init(): Promise<void> {
 		</a>
 	);
 
-	const navigationBar = (await elementReady('.UnderlineNav-body'))!;
 	navigationBar.append(link);
 	navigationBar.replaceWith(navigationBar);
 
 	select('.js-responsive-underlinenav .dropdown-menu ul')!.append(
-		createDropdownItem('Gists', href),
+		createDropdownItem('Gists', getUser().url),
```

Comment:
should be `user.url`

created PR:

#6050 Update profile-gists-link.tsx - use variable user